### PR TITLE
GN-36 Print styling

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/GN-36

### CODE REVIEW

File: webpack.config.prod.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = function(env) {
 		plugins: [
 			new webpack.BannerPlugin({
 				banner: [
-					`NICE Global Nav ${env.version || pkg.version} | ${
+					`NICE Global Nav ${pkg.version} | ${
```
Comment (greypiglet) [inline webpack.config.prod.js]: @ediblecode I took the env out as it wasn't building - not sure under what circumstance the env version would need to be different from the code? Will replace if necessary!

Comment (ediblecode) [reply webpack.config.prod.js]: it's because the build numbers on TeamCity e.g. 1.2.3-rabc1234 aren't compatible with npm package versions

Comment (ediblecode) [reply webpack.config.prod.js]: there's a section in the readme that rereferences it: https://github.com/nice-digital/global-nav#production-build

**Comment by greypiglet** [comment]
@ediblecode I think this is ready to go if you get a chance to look

**Pull request merged**